framework/include/utils/InputParameters.h

```diff
@@ -754,7 +773,7 @@ class InputParameters : public Parameters
     bool _is_private = false;
     bool _have_coupled_default = false;
     /// The default value for optionally coupled variables
-    Real _coupled_default = 0;
+    std::vector<Real> _coupled_default = {};
```

Comment:
This looks weird. The default is an empty array? Should not that be an array with one zero in it? That would correspond to the original implementation IMO.